solution.txt

```diff
@@ -178,10 +161,8 @@ const numArrays = [
 ];
 
 // Complete Exercise 13 below...
+let numList = numArrays.slice(-1);
```

Comment:
nice! this negative numbers trick is really helpful for easily getting the last element in an array